tools/cldr-apps/src/main/webapp/surveytool.css

```diff
@@ -1702,6 +1703,13 @@ The the following can work here:
 
 */
 
+/* Kashmiri and Urdu are typically in Nastaliq. Kashmiri in particular uses a letter that is 
+   not rendered correctly in Noto Naskh (see CLDR-16595) */
+
+lang(ks), lang(ur) {
+	font-family: "Noto Nastaliq Urdu";
+}
+
```

Comment:
maybe you meant "I didn't put `.cldrValue` here in conjunction with the `lang()` …"  …  this is 💯

Reply:
I did initially, I then overwrote it, because at least in the side panel there is no `.cldrValue`